src/screens/SignInV2/elements/BottomOverlay/index.js

```diff
@@ -143,7 +142,7 @@ const bottomOverlayStyles = StyleSheet.create({
     fontFamily: fonts.inter[700]
   },
   link: {
-    color: colors.blue,
+    color: COLORS.blue,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the link has been changed from `colors.blue` to `COLORS.blue`. Ensure that this change is intentional and consistent with the overall theme of the application.
```diff
-    color: colors.blue,
+    color: COLORS.blue,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->